linkage/pvs_like_case_study/README.md

```diff
@@ -36,8 +38,36 @@ $ Rscript -e "renv::restore(library=.libPaths())"
 Updating R packages should be done within the environment, installing them
 and then calling `renv::snapshot(type = "all")`.
 
+### Local Spark version
+
+Unfortunately, it isn't possible to install Spark with conda.
+Instead, I have used a Singularity image with Spark, and then activated
+a conda environment inside it.
+
```

Comment:
It looks like you start with a Docker image, which is made into a singularity image. I assume there is still a desire for step implementations like this one running with Docker and singularity. Is that correct?

Reply:
I only did that for convenience, since it is easy to get Docker images from Docker Hub.

I never ran any of this using Docker, but I think we will want that to work, yes.

Does that answer your question? Not sure I completely understood it.